components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Model/MonitoredItemModelTestHelper.cs

```diff
@@ -0,0 +1,313 @@
+﻿namespace Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Tests.Model {
+    using Microsoft.Azure.IIoT.OpcUa.Core.Models;
+    using Microsoft.Azure.IIoT.OpcUa.Protocol.Models;
+    using System;
+    using System.Collections.Generic;
+
+    internal static class MonitoredItemModelTestHelper {
```

Comment:
Great that you added the test! Thanks! 

I'm not sure why we need this class however. It does the same checks that IsSameAs implemented on the model classes does? Those are sufficient. Shouldn't have to duplicate that code here? It's not used outside the test project.


This class we shouldn't have to have however. Instead, can we call the IsSameAs on the actual objects instead? So that our tests actually verify those and not this helper.

Reply:
Well, I was thinking about having two parallel ways of comparing the objects. But I see your point, maybe they are overkill.

Reply:
Removed the separate IsSameAs in test project